lib/package.gi

```diff
@@ -1210,6 +1210,18 @@ InstallGlobalFunction( DirectoriesPackagePrograms, function( name )
       # This package is not known.
       return [];
     fi;
+# FIXME: inject another secondary path??? to allow "out of tree builds"
+# for packages, and/or allow GAP itself to store binaries for the package
+# so that the precise combination of GAP and package are matched...
+# That is, take the precise GAP version (possibly including the git commit
+# it was compiled from), plus the package version (and possibly also the locations of both?)
+# and combine them into a unique ID (e.g. via a hash), then use that to
+# store .so binaries
+
+
+# also add a new mechanism to integrate kernel extensions into a package:
+# let the package declare in its PackageInfo.g that it has a kernel extension;
+# how to build it; what's its name; then GAP can take care of that dynamically
```

Comment:
Can the intended new entries in `PackageInfo.g` also replace the `prerequisites.sh` files used by`BuildPackages.sh`?

Reply:
Hopefully / perhaps -- it depends a bit on what the package does in there...